src/commands.rs

```diff
@@ -2140,3 +2163,22 @@ impl ToRedisArgs for LposOptions {
         false
     }
 }
+
+/// Enum for the LEFT | RIGHT args used by some commands
+pub enum Direction {
+    Left,
+    Right,
+}
+
+impl ToRedisArgs for Direction {
+    fn write_redis_args<W>(&self, out: &mut W)
+    where
+        W: ?Sized + RedisWrite,
+    {
+        let s = match self {
```

Comment:
```suggestion
        let s: &[u8] = match self {
```
Seems to be some compilation errors.